src/events/mouse.js

```diff
@@ -731,6 +731,11 @@ p5.prototype._onmouseup = function(e) {
   const context = this._isGlobal ? window : this;
   let executeDefault;
   this._setProperty('mouseIsPressed', false);
+
+  if (this.touchend) {
```

Comment:
Does this get called implicitly because the browser triggers a mouse event after the touch event? It'd be helpful adding a comment here explaining if so.

Reply:
I'll add a comment for this and the other PR

Reply:
I've added the comment